### PR TITLE
Update NLCD data for 2019

### PR DESCRIPTION
**Pull request by timstallmann**

### CODE REVIEW

File: metric/blockgroup/dPCTIMP.csv [hunk 1]
```diff
@@ -1,153 +1,154 @@
-id,y_2001,y_2006,y_2011,y_2016
```
Comment (libbymcclure) [inline metric/blockgroup/dPCTIMP.csv:1]: Just a note that git isn't highlighting new column (2019 year) of data on this file

File: metric/blockgroup/rPCTIMP.csv [hunk 2]
```diff
@@ -1,153 +1,154 @@
-id,y_2001,y_2006,y_2011,y_2016
```
Comment (libbymcclure) [inline metric/blockgroup/rPCTIMP.csv:1]: same as above

File: metric/blockgroup/dPCTIMP.csv (hunk 1 above)
Comment (timstallmann) [reply metric/blockgroup/dPCTIMP.csv:1]: oh weird! it is for me.. wonder what's up with that.

**Review by libbymcclure — COMMENTED** [review]
Looks good to me! Just a note that sometimes Git isn't highlighting individual character changes, but we've talked about that.

**Comment by timstallmann** [comment]
Thx @libbymcclure! :stars:

**Pull request merged**